lib/ts/recipe/session/index.ts

```diff
@@ -83,14 +83,6 @@ export default class SessionWrapper {
     static updateJWTPayload(sessionHandle: string, newJWTPayload: any) {
         return Recipe.getInstanceOrThrowError().recipeInterfaceImpl.updateJWTPayload({ sessionHandle, newJWTPayload });
     }
-
-    static verifySession = (options?: VerifySessionOptions) => {
```

Comment:
This also needs to be exposed as a static function inside the Wrapper class

Reply:
done